README.md

```diff
@@ -3,7 +3,7 @@
 
 _Reache_ is a desktop app that helps **busy working professionals** manage their large list of contacts by providing an easy-to-use interface to store contacts and organize meetings. _Reache_ is especially suited for professionals who would like to optimize their contact management by using a **Command Line Interface (CLI)**.
 
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/NewUI.png)
```

Comment:
I think the name of this file should be `Ui.png` so that it can be detected by the scripts for the module website.

Reply:
Alright will change it